### PR TITLE
Broken Links fix for 2696

### PR DESCRIPTION
**Pull request by ramnar**
The following broken links are fixed in the community documentation

FILE: ./sig-azure/charter.md
[✖] contributors/devel/owners.md → Status: 400 Error: ENOENT: no such file or directory, access '/home/jorge/src/kubernetes/community/sig-azure/contributors/devel/owners.md'

FILE: ./sig-storage/contributing.md
[✖] https://git.k8s.io/kubernetes/examples/volumes/flexvolume → Status: 404

FILE: ./sig-multicluster/CONTRIBUTING.md
[✖] http://github.com/kubernetes/community/blob/master/contributors/design-proposals/sig-multicluster → Status: 404
[✖] http://github.com/kubernetes/community/blob/master/contributors/design-proposals/sig-multicluster/template.md → Status: 404

FILE: ./contributors/guide/github-workflow.md
[✖] https://github.com/$user/kubernetes → Status: 404
[✖] https://reviewable.k8s.io → Status: 0 Error: getaddrinfo ENOTFOUND reviewable.k8s.io reviewable.k8s.io:443

### CODE REVIEW

File: contributors/guide/github-workflow.md [hunk 1]
```diff
@@ -220,7 +220,7 @@ fork.
 
 Very small PRs are easy to review.  Very large PRs are very difficult to review.
 At the assigned reviewer's discretion, a PR may be switched to use
```
Comment (cblecker) [inline contributors/guide/github-workflow.md]: Please remove everything in this paragraph other than `Very small PRs are easy to review.  Very large PRs are very difficult to review.`. We don't use reviewable at all anymore.

File: sig-azure/charter.md [hunk 2]
```diff
@@ -97,4 +97,4 @@ Issues impacting multiple subprojects in the SIG should be resolved by SIG Techn
 [super-majority]: https://en.wikipedia.org/wiki/Supermajority#Two-thirds_vote
 [KEP]: https://github.com/kubernetes/community/blob/master/keps/0000-kep-template.md
 [sigs.yaml]: https://github.com/kubernetes/community/blob/master/sigs.yaml#L1454
-[OWNERS]: contributors/devel/owners.md
+[OWNERS]: OWNERS
```
Comment (cblecker) [inline sig-azure/charter.md]: should be `/contributors/guide/owners.md`

File: contributors/guide/github-workflow.md (hunk 1 above)
Comment (ramnar) [reply contributors/guide/github-workflow.md]: this is done.thanks for the review

File: sig-azure/charter.md (hunk 2 above)
Comment (ramnar) [reply sig-azure/charter.md]: this is done.thanks for the review

**Pull request merged**